_health-care/_js/components/form-elements/ErrorableRadioButtonGroup.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A form radio button group with a label that can display error messages.
+ *
+ * Validation has the following props.
+ * `errorMessage` - Error string to display in the component.
+ *                  When defined, indicates group has a validation error.
+ * `label` - String for the group field label.
+ * `options` - Array of options to populate group.
+ * `required` - boolean. Render marker indicating field is required.
+ * `value` - string. Value of the select field.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableRadioButtonGroup extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = _.uniqueId('errorable-radio-button-group-');
+  }
+
+  handleChange(domEvent) {
+    // hackery to allow boolean values to work with radio buttons
+    const formValue = domEvent.target.value;
+    let storeValue = formValue;
+    if (formValue && typeof formValue === 'string') {
+      if (formValue.toLowerCase() === 'true') storeValue = true;
+      if (formValue.toLowerCase() === 'false') storeValue = false;
+    }
+    this.props.onValueChange(storeValue);
+  }
+
+  render() {
+    const options = _.isArray(this.props.options) ? this.props.options : [];
+    const storedValue = this.props.value;
+    let reactKey = 0;
+    const optionElements = options.map((obj) => {
+      let optionLabel;
+      let optionValue;
+      if (_.isString(obj)) {
+        optionLabel = obj;
+        optionValue = obj;
+      } else {
+        optionLabel = obj.label;
+        optionValue = obj.value;
+      }
+      let checked;
+      checked = (storedValue && optionValue.toString() === storedValue.toString()) ? checked = 'checked=true' : '';
```

Comment:
This is a little wonky. Why do you have to convert both of these to strings? They should both be stored as booleans, so you should be able to just compare them. I think all you need to replace lines 51-52 is:
```javascript
const checked = storedValue !== undefined && optionValue === storedValue ? 'checked=true' : '';
```

I'm assuming you have the initial `storedValue` check to make sure it's not undefined? Unfortunately we have to actually check if it's not undefined because simply calling it will return it's value, which, if set, is either true or false, which will affect what this whole expression evaluates to.